Application/EdFi.Ods.Api/Conventions/TemplateSetConventions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.Ods.Api.Conventions
+{
+    public static class TemplateSetConventions
```

Comment:
"TemplateSet" is terminology copied in from the CodeGen utility. I think `PluginConventions` would be a better name for this class in this context.

Reply:
Fixed it . Thanks